databroker/tests/test_urls.py

```diff
@@ -10,14 +11,18 @@
                           ('run_stop', 'find_run_stops'),
                           ('event', 'find_events'),
                           ('event_descriptor', 'find_descriptors')])
+
+@pytest.mark.skipif(os.environ.get('INCLUDE_V0_SERVICE_TESTS') != '1',
```

Comment:
You can put this at the module level so just 

```python
pytest.mark.skipif(...)
```

which will skip the whole module.

Reply:
updated

Reply:
For future referenece, it is 

```python
pytestmark = pytest.mark.skipif(...)
```